test/functionalTest/cases/0000_xml_parse/xmlParsePostNotifyContext.test

```diff
@@ -52,9 +52,9 @@ payload='<?xml version="1.0" encoding="UTF-8"?>
                 <value>1110</value>
               </contextMetadata>
               <contextMetadata>
-                <name></name>
-                <type></type>
-                <value></value>
+                <name>id2</name>
```

Comment:
XML is now a deprecated feature, so modifications in XML-based .test are not very relevant. However, I wonder why this has changed... there was any bug in the XML metadata rendering?